product_dimension_compute_volume/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Product Dimension Compute Volume",
+    "summary": """
+        Automatically compute the volume depending on the dimensions.""",
+    "version": "12.0.1.0.0",
+    "category": "Product",
+    "website": "https://coopiteasy.be",
+    "author": "Odoo Community Association (OCA), Coop IT Easy SCRLfs",
+    "maintainer": "Coop IT Easy SCRLfs",
```

Comment:
Afaik this key is unused.

Reply:
OK! I'll remove this.

Reply:
Hello, the key is used, but it should be set if you have intentions of maintaining the module.
You set your github login and AFAIR it will allow you to merge/handle PRs related to your module. So you can work on it without beeing member of a PSC.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_maintainer_role.rst#maintainer-role-and-responsibilities